cli/server.ts

```diff
@@ -23,7 +23,7 @@ let userProjectsDir = path.join(process.cwd(), userProjectsDirName);
 let docsDir = ""
 let tempDir = ""
 let packagedDir = ""
-let localHexDir = path.join("hexcache");
+let localHexDir = path.join("built", "hexcache");
```

Comment:
No, in this file, "built" is a hard-coded string everywhere, see the 2 following functions
https://github.com/Microsoft/pxt/blob/master/cli/server.ts#L53
https://github.com/Microsoft/pxt/blob/master/cli/server.ts#L114